### PR TITLE
LRDOCS-7777 Learning to Rank Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-7777
https://issues.liferay.com/browse/LRDOCS-8505

@rbohl
@lipusz

- LRDOCS-7777 initial content port
- LRDOCS-7777 nav work
- LRDOCS-7777 remove @product@ tokens
- LRDOCS-7777 image
- LRDOCS-7777 updates, edits
- LRDOCS-8505 port the LES overview content into the LES landing page
- LRDOCS-7777 pare down and refine the intro
- LRDOCS-7777 a bit more refinement
- LRDOCS-7777 Wordsmithing.

### CODE REVIEW

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+LES Learning to Rank requires a Liferay Enterprise Search subscription. It's important to understand that the [Elasticsearch Learning to Rank plugin](https://elasticsearch-learning-to-rank.readthedocs.io/en/latest/index.html) is not produced by Elastic, and there is not a pre-built plugin for all of Liferay's supported Elasticsearch versions. See the [LES Compatibility Matrix for details](https://help.liferay.com/hc/en-us/articles/360016511651#Liferay-Enterprise-Search).
+
+## Disabling Learning to Rank on a Search Page
+
+Learning to Rank does not work with the [Sort widget](../search-pages-and-widgets/search-results/sorting-search-results.md).
+
+If LES Learning to Rank is deployed, but you must disable it on a particular Search page (perhaps to use the Sort widget),
+
+1. Add a [Low Level Search Options](../search-pages-and-widgets/understanding-low-level-search-options.md) widget to the Search page.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md:17]: @rbohl this link is broken - is the article is points to still pending or should it point to something else?

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+## Step 2: Training and Uploading a Model
+
+Detailed instructions on training models is outside the scope of this guide. This requires the intervention of data scientists, who can recommend appropriate tools and models. Use what works for you. In doing so, you'll almost certainly be compiling [Judgment lists](https://elasticsearch-learning-to-rank.readthedocs.io/en/latest/core-concepts.html#judgments-expression-of-the-ideal-ordering) and [feature sets](https://elasticsearch-learning-to-rank.readthedocs.io/en/latest/building-features.html) that can be used by the training tool you select to generate a model that produces good search results. Once you have a model, upload it to the Learning to Rank plugin.
+
+## Step 3: Upload the Model to the Learning to Rank Plugin
+
+You'll upload the model using a `POST` request, but first you need to make sure you have a `_ltr` index and a feature set uploaded to the Learning to Rank plugin. Use Kibana (via the [LES Monitoring widget](./monitoring-elasticsearch.md)), to make these tasks easier.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md:98]: @rbohl - just confirming - the monitoring link is intentionally broken as that's pending content, right?

Comment (jrhoun) [reply docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md:98]: Never mind, just saw https://github.com/jrhoun/liferay-learn/pull/911.

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md:17]: it should be fixed if/when this gets merged (if it's not renamed or re-orged): https://github.com/sez11a/liferay-learn/pull/359

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md (hunk 2 above)
Comment (rbohl) [reply docs/dxp/7.x/en/using-search/liferay-enterprise-search/learning-to-rank.md:98]: yes, monitoring is also in your queue

**Comment by jrhoun** [comment]
Merged. Thanks!

**Pull request closed without merging**